plugins/uniques.js

```diff
@@ -154,8 +182,31 @@ window.plugin.uniques.onPublicChatDataAvailable = function(data) {
 		    // search for "Your Portal x neutralized by y"
 		    // search for "Your Portal x is under attack by y"
 			var portal = markup[1][1];
-			var guid = window.findPortalGuidByPositionE6(portal.latE6, portal.lngE6);
-			if(guid) plugin.uniques.setPortalVisited(guid);
+			match = true;
+			plugin.uniques.setPortalAction(portal,'visited');
+		} else if (plext.plextType == 'SYSTEM_NARROWCAST'
+			&& markup.length==3
+			&& markup[0][0] == 'TEXT'
+			&& markup[0][1].plain == 'You claimed Scout Controller on '
+			&& markup[1][0] == 'PORTAL') {
+				// search for "You claimed Scout Controller on "
+				var portal = markup[1][1];
+				match = true;
+				plugin.uniques.setPortalAction(portal,'scouted');
+		} else if (plext.plextType == 'SYSTEM_NARROWCAST'
+			&& markup.length==3
+			&& markup[0][0] == 'TEXT'
+			&& markup[0][1].plain == 'You were displaced as Scout Controller on '
+			&& markup[1][0] == 'PORTAL') {
+				// search for "You were displaced as Scout Controller on"
+				var portal = markup[1][1];
```

Comment:
Indentation of `&&` does not much similar cases in upper code

Reply:
now matching the same broken indentation as the upper code